corehq/apps/smsbillables/migrations/0005_smsbillable_twilio_gateway_fee.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsbillables', '0004_bootstrap_smsgh_rates'),
+    ]
+
+    operations = [
```

Comment:
FYI I ran that locally and got the following:
```
BEGIN;
ALTER TABLE "smsbillables_smsbillable" ADD COLUMN "twilio_gateway_fee" numeric(10, 4) NULL;
ALTER TABLE "smsbillables_smsbillable" ALTER COLUMN "twilio_gateway_fee" DROP DEFAULT;
COMMIT;
```
I don't know why Django insists on the `DROP DEFAULT` but as far as I can tell that's a no-op since the column isn't created with a default value.
So I think this is GTG.